modules/AOS_Invoices/language/en_us.lang.php

```diff
@@ -201,6 +201,11 @@
     'LBL_SHIPPING_TAX_USDOLLAR' => 'Shipping Tax (Default Currency)',
     'LBL_SHIPPING_TAX_AMT_USDOLLAR' => 'Shipping Tax (Default Currency)',
     'LBL_GRAND_TOTAL_USDOLLAR' => 'Grand Total (Default Currency)',
-    'LBL_INVOICE_TO' => 'Invoice To'
+    'LBL_INVOICE_TO' => 'Invoice To',
+    'LBL_DISCOUNT__AMOUNT_USDOLLAR' => 'Discount Amount (Default Currency)',
```

Comment:
In this PR i did not changed at all existed LBL pointers, please raise suggestion in separate issue.